triangle.py

```diff
@@ -0,0 +1,25 @@
+import math
+
+class Triangle(): 
+	start_final: float
+	start_point: float
+	final_point: float
+	area: float
+	height: float
+
+	def __init__(self, start_final, start_point, final_point):

```

Comment:
I think you should add types for the arguments to this function as well as for the return.

Reply:
added types, this function doesn't return so I'll omit a return type